lib/aws_auditor/scripts/audit.rb

```diff
@@ -9,14 +9,22 @@ class << self
         attr_accessor :options
       end
 
-      def self.execute(args_array, options=nil)
-        aws(args_array.delete_at(0))
-        tag_name = args_array.delete_at(0) || "no-reserved-instance"
+      def self.execute(environment, options=nil)
+        aws(environment)
         @options = options
+
+        if options[:custom_tag] && !options[:no_tag]
+          tag_name = Output.ask("Enter custom tag: "){ |q|  q.validate = /\S+/ }
```

Comment:
Did you consider passing the tag name value as a argument instead of an argument to ask for it?